src/types/common.ts

```diff
@@ -182,6 +182,13 @@ export type RoleReactionConfigResponse = {
   success: boolean
 }
 
+export type ReactionRoleListConfigItem = {
+  role?: string
+  emoji?: string
+  url?: string
+  title?: string
+}
+
```

Comment:
all API repsonse models should be generated inside `api.ts`
-> remove this
-> remove all reactions role models inside `common.ts`
-> update models in reaction role commands